### PR TITLE
FlattenDictionary addition to Community Common Scripts

### PR DESCRIPTION
**Pull request by Ext-Defender**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Adds a script to the Community Common Scripts pack that flattens JSON dictionaries.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CommunityCommonScripts/FlattenDictionary/FlattenDictionary.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            items.append((n_key, v))
+        
+    return dict(items)
+            
+''' MAIN FUNCTION '''
+
+
+def main():
+    try:
+        data = json.loads(demisto.args()["unflatten_dict"])
```
Comment (mmhw) [inline Packs/CommunityCommonScripts/FlattenDictionary/FlattenDictionary.py:34]:
```suggestion
        data = json.loads(demisto.args()["unflattened_dict"])
```
Note that it should be as defined in the yal file

**Review by mmhw — COMMENTED** [review]
Hi @contributor_user,
Thank you for your contribution!
Good work :)

Some Comments:
1. Please bump the pack version in the pack_metadata.json file and add notes describing your changes to a new Release Notes file.
To do that, please run the `demisto-sdk update-release-notes  -i Packs/CommunityCommonScripts` command - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.
2. It would be nice if you could add a unit test for your new command.
You can use the following guide - https://xsoar.pan.dev/docs/integrations/unit-testing as well as examples of unit tests in the HelloWorld integration and other packs from the content repository.
3. To add outputs to Context Data the flatten_dict function will have to return CommandResults, see [here](https://xsoar.pan.dev/docs/integrations/code-conventions#commandresults).

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

File: Packs/CommunityCommonScripts/FlattenDictionary/FlattenDictionary.py (hunk 1 above)
Comment (Ext-Defender) [reply Packs/CommunityCommonScripts/FlattenDictionary/FlattenDictionary.py:34]: Thank you, I've made the change.

**Review by mmhw — COMMENTED** [review]
Good job, note two changes

**Comment by mmhw** [comment]
Hi @Ext-Defender , 
Thank you for your contribution. 
I have not received a response from you for a long time, so I will close your PR now.
Please feel free to re-open it when you are available to continue. 
Thanks again.

**Pull request closed without merging**